### PR TITLE
Replaced all humbug_get_contents with the namespaced one

### PR DESCRIPTION
**Pull request by waltertamboer**
This PR fixes issue #45

### CODE REVIEW

File: src/Strategy/GithubStrategy.php [hunk 1]
```diff
@@ -68,7 +68,7 @@ public function download(Updater $updater)
     {
         /** Switch remote request errors to HttpRequestExceptions */
         set_error_handler(array($updater, 'throwHttpRequestException'));
-        $result = humbug_get_contents($this->remoteUrl);
+        $result = \Humbug\get_contents($this->remoteUrl);
```
Comment (theofidry) [inline src/Strategy/GithubStrategy.php]: could you import it? `use function Humbug\get_contents()`, same for the other places

**Comment by waltertamboer** [comment]
Fixed, Travis build is succeeding so not sure what you mean with the prefer lowest failing?

**Comment by theofidry** [comment]
Hm somehow it gets the 1.1.0 even if prefer lowest 🤔

**Comment by waltertamboer** [comment]
Thanks! Will require the dev-master for now so at least I can finish my project :)

**Comment by wapmorgan** [comment]
What about releasing a new version with these changes?

**Comment by theofidry** [comment]
It will be released when I'll find the time to do a round of phar-updater, either require `dev-master` or that specific commit if that's urgent meanwhile

**Comment by padraic** [comment]
We can release a new version, I think, for this change. It will get is over the deprecation notice for current usage while the other more significant PRs are being reviewed.

**Comment by wapmorgan** [comment]
I hope. Because currently it doesn't build out-of-the-box due to this notice.

**Pull request merged**